src/forms/formHelpers.js

```diff
@@ -1,33 +1,75 @@
-/** @module formHelpers
- * @todo Separate into better-named dirs/files
- */
+/** @module formHelpers */
+// @todo Separate into better-named dirs/files
 
 // REACT COMPONENTS
 import React from 'react';
 import { Icon } from 'semantic-ui-react';
 
 // ========================================
-// INPUT CONTAINER COMPONENTS
+// FEEDBACK FOR USER
 // ========================================
 
-/** Red bold text that only appears if `validRow` is `false` */
-const InvalidMessage = function ({ validRow, className, children, ...otherProps }) {
+/** Puts a message above the given children.
+ * 
+ * @description It should always be the only item in its
+ *     row - block content. Currently meant to be used with
+ *     an input row to give short instructions or user error
+ *     message. It also attempts to conform to accessibility: 
+ *     @see {link https://www.w3.org/WAI/tutorials/forms/instructions/#using-aria-labelledby}. 
+ *     It's given a hook to allow an input to label it as a
+ *     description with aria. `tab-index` for IE (see 'note').
+ * 
+ * @param {string|object} children Objects react can render.
+ * @param {string} ariaName ID of node this message will be
+ *     describing.
+ * @param {string} [className] Optional classes to add to allow
+ *     access with css.
+ * @param {string|object|null} [message] To be rendered above
+ *     the children.
+ * @param {boolean} [isUserError] Is this an error message?
+ *     Another option would be `type` instead - descriptor(s)
+ *     for the message that will currently be turned into
+ *     classes. Things like 'invalid'.
+ *
+ * @returns React element
+ */
+const WithMessageAbove = function ({ children, ariaName, isUserError, message, className }) {
```

Comment:
I like that you've added the ariaName to make it easy to associate with another node.